### ISSUE TITLE
[Feature Request] Mouse scroll-mode modifier

### ISSUE THREAD

**Issue by DarkFox** [fifafu/BetterTouchTool#304]
I use a Logitech Marble mouse (trackball), and before macOS Sierra, I used Karabiner to make holding one of the buttons on it, turn the trackball into a scroll wheel.

Unfortunately, Karabiner isn't compatible with Sierra, so now I'm looking to my other favourite input-device-modifying application for help.

I see how this would mainly apply to trackball users who don't have a scroll wheel, but I could imagine it'd benefit others with limited input devices, or those who have limited finger dexterity, or similar needs.

tl;dr: Hold button/key = Mouse input scrolls instead of moving cursor.

**Comment by fifafu** [comment]
Sorry for not replying earlier to this issue. I'm currently cleaning up here, which is why I'm closing some older issues. In case you still have questions or problems, please create a new issue - I'm now much more responsive than a few months ago.

This is definitely something for Karabiner Elements instead of BTT because it would require to stop the mouse cursor from moving - which requires deeper system integration than BTT does.
